integration-test/src/test/java/org/apache/iotdb/confignode/it/cluster/IoTDBClusterNodeErrorStartUpIT.java

```diff
@@ -240,7 +247,8 @@ public void testIllegalNodeRestart()
       // Shutdown and check
       EnvFactory.getEnv().shutdownConfigNode(1);
       EnvFactory.getEnv().shutdownDataNode(0);
-      while (true) {
+      int retryTimes;
+      for (retryTimes = 0; retryTimes < maxRetryTimes; retryTimes++) {
```

Comment:
These codes has been merged into master. Maybe this branch should merge with master.

Reply:
These codes are cherry-picked from the master